src/components/PingForm.vue

```diff
@@ -1,25 +1,148 @@
 <template>
-    <h1>Ping查询</h1>
+  <div id="pingform">
```

Comment:
命名规范：
1. HTML 中不区分大小写，所以 tag 名、属性名都用 kebab-case，id 和 class 的值也用纯小写，但是用 `-` 或者 `_` 来分割单词
2. JS 中用 camelCase，不用 - 和 _

```suggestion
  <div id="ping-form">
```